docs/api/python/python.rst

```diff
@@ -1,4 +1,4 @@
-Python API Documentation
+Python API
```

Comment:
Should the visibility of the individual api toctrees really be hidden?
Also: `compatibility API` -> `Compatibility Python API` and `regular Python API` -> `Regular Python API`
Can you make the former change also for the header of the compat api page?